addon/templates/components/polaris-breadcrumbs.hbs

```diff
@@ -1 +1,30 @@
-{{polaris-breadcrumbs/breadcrumb breadcrumb=breadcrumb}}
+{{#if breadcrumb.url}}
+  {{#polaris-unstyled-link
+    key=breadcrumb.content
```

Comment:
IIRC this `key` property is a [React](https://reactjs.org/docs/lists-and-keys.html) thing so I don't think we need it